### DISCUSSION TITLE
LGTM画像 テスト

### DISCUSSION THREAD

**Discussion by siso001** [lgtm-factory/lgtm-factory#40] [Scrap]
[![LGTMeow](https://lgtm-images.lgtmeow.com/2022/04/27/16/9f452c98-7d03-4055-8b19-5822fc9a1452.webp)](https://lgtmeow.com)

**Comment by siso001** [comment]
https://lgtmeow.com/

**Reply by kazzyfrog** [reply]
上記のサービスの特徴：
- TOPページに、LGTM画像の一覧が表示されている
- LGTM画像をクリックすると、GitHubのマークダウンで使える形式の画像URLが、コピーできる
- →GitHub上で、使える！サイトを訪れて、１タッチ！めっちゃ簡単！

その他：
- 一覧画像のリフレッシュ機能
- ランダムコピー機能
- 画像アップロード機能

**Comment by siso001** [comment]
[![LGTMeow](https://lgtm-images.lgtmeow.com/2023/10/24/11/18eb7c36-8bbd-4241-8d3f-c3164493fcdc.webp)](https://lgtmeow.com)

**Comment by kazzyfrog** [comment]
テスト🐈

[![LGTMeow](https://lgtm-images.lgtmeow.com/2023/03/12/23/9f0ea907-4b8c-4fb4-b04f-3a23ba2d5360.webp)](https://lgtmeow.com)
